internal/page/dashboard.go

```diff
@@ -56,6 +47,20 @@ func Dashboard(tmpl template.Template, donorStore DonorStore, dashboardStore Das
 			tabCount++
 		}
 
+		if r.Method == http.MethodPost {
+			lpa, err := donorStore.Create(r.Context())
```

Comment:
Would it be simpler to add the details to the new LPA at this point vs. doing `.Latest` in all the handlers? We could even have a `donorStore.CreateFromLatest` or something, or just make it check in `.Create` whether one exists?

Reply:
Much, I'll move to here.